projects/sm-pwr/Untitled.ipynb

```diff
@@ -0,0 +1,110 @@
+{
```

Comment:
<p>delete this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dpploy/engy-4390/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>